### DISCUSSION TITLE
Can Shiny.net libraries be used to Send push notifications to Firebase from Windows?

### DISCUSSION THREAD

**Discussion by exquirentibus** [shinyorg/shiny#1387] [Q&A]
I've had great success using Shiny.net to receive Firebase Push Notifications on IOS and Android!!

Great libraries!!!

I've been using RestSharp or Postman to send on Windows using the soon to be deprecated Firebase API.

Is there a way to send a Push to Firebase using Shiny.net on Windows?

And if so, will it automatically handle the new V1 API and the slight format changes between IOS and Android?

**Comment by aritchie** [comment]
Glad to hear you've had success using Shiny.

Shiny client side does not support windows

There is a server library that will be delivered in the future that allows you to send push notifications to the native APIs (firebase for android, apple for iOS/maccatalyst), but I don't have a timeline for when/if that will be released.
